.golangci.yaml

```diff
@@ -107,8 +107,17 @@ issues:
   fix: true
   max-same-issues: 0
   exclude-rules:
+  - linters:
+    - ineffassign
+    text: "ineffectual assignment" # ignore err not checked in test files
+    path: test\.go
```

Comment:
For this particular exception I'd be inclined to add it inline. If we add this here we might end up in ineffectual assignments across the codebase (which is a thing I once cleaned up already and I'd prefer to not need to do it again).